config.json

```diff
@@ -836,6 +836,17 @@
         "Maybe"
       ]
     },
+    {
+      "uuid": "d1dc946d-a142-42ed-abec-e53605635c0d",
+      "slug": "twelve-days",
+      "core": false,
+      "unlocked_by": null,
+      "difficulty": 4,
+      "topics": [
+        "algorithms",
+        "text_formatting"
```

Comment:
> If the algorithms is to be added, I will want all exercises that should have the topic to have it. Since currently none do (but I assume some should, if it exists), I would ask for either:
> 
> * Add it to all exercises that should have it in a separate PR or a separate commit in this PR, then keep it in this PR.
> * Do not add it in this PR, but after this PR is merged submit another PR that does add it.
> * Do not add it in this PR, and do nothing further.

same (but text_formatting instead of algorithms)